test/src/sdk-harness/test_projects/hashing/mod.rs

```diff
@@ -468,11 +468,21 @@ mod sha256 {
             Hash::Sha256,
         ));
 
-        let call_1 = instance.methods().sha256_struct(true).call().await.unwrap();
-        let call_2 = instance.methods().sha256_struct(true).call().await.unwrap();
+        let call_1 = instance
+            .methods()
+            .sha256_str_arrayuct(true)
```

Comment:
`sha256_str_arrayuct ` looks like a typo (here and several instances below)

Reply:
Fixed.

Reply:
Fixed.